packages/mdc-segmented-button/segment/_segment-theme.scss

```diff
@@ -0,0 +1,157 @@
+//
+// Copyright 2020 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+// Theme: Variables, functions, and mixins required to theme component segment
+
+// TODO: is it ok to use mdc-button?
+@use "@material/button";
+@use "@material/theme/variables" as theme-variables;
+@use "@material/feature-targeting/functions" as feature-targeting-functions;
+@use "@material/feature-targeting/mixins" as feature-targeting-mixins;
+@use "@material/typography/mixins" as typography-mixins;
+
+$unselected-ink-color: rgba(theme-variables.prop-value(on-surface), 0.6);
+$unselected-outline-color: rgba(theme-variables.prop-value(on-surface), 0.12);
+$unselected-container-fill-color: rgba(theme-variables.prop-value(surface), 1);
+
+$selected-ink-color: rgba(theme-variables.prop-value(primary), 1);
+$selected-outline-color: rgba(theme-variables.prop-value(primary), 1);
+$selected-container-fill-color: rgba(theme-variables.prop-value(primary), 0.08);
+
+$height: 36px;
+$min-width: 48px;
+$end-padding: 12px;
+$border-width: 1px;
+$border-radius: 4px;
+$icon-width: 24px;
+$icon-font-size: 18px;
+$label-padding: 6px;
+
+@mixin base($query) {
+  $feat-structure: feature-targeting-functions.create-target($query, structure);
+  $feat-typography: feature-targeting-functions.create-target(
+    $query,
+    typography
+  );
+
+  @include typography-mixins.typography(button, $feat-typography);
+
+  @include feature-targeting-mixins.targets($feat-structure) {
+    display: inline-flex;
+    vertical-align: top;
+    align-items: center;
+    height: $height;
+    min-width: $min-width;
+    padding: 0px;
+    padding-left: $end-padding;
+    padding-right: $end-padding;
+    border-width: $border-width;
+  }
+}
+
+@mixin unselected($query) {
```

Comment:
This is not a theme mixin which can be used by clients. Move it to `_segmant.scss`.

Same below.